### ISSUE TITLE
Link para termos de uso

### ISSUE THREAD

**Issue by rodrigondec** [idvogados/backend#5]
O link para os termos de uso deverá ficar no `README.md` ou no `CONTRUBUTING.md`?

Após a discussão tida nesse issue, foi decidido que ficará no `README.md` .

**Comment by pablohildo** [comment]
Acho importante que fique no `README.md`.
Conhecer os termos é válido pra qualquer pessoa, não só quem for de fato contribuir com o repositório.
Além disso, sua visibilidade (maior no `README.md`) é valiosa do ponto de vista de transparência.

**Comment by pabrrs** [comment]
Concordo com o colega @pablohildo, no `README.md` tem maior visibilidade.
